qemu/tests/eject_media.py

```diff
@@ -1,12 +1,13 @@
 import logging
 import time
 
-from autotest.client.shared import error
+from avocado.core import decorators
```

Comment:
Please directly import `fail_on` from `avocado`, because `avocado.core` is not a public module and should not be used by test script.